spec/helpers/tasklist_helper_spec.rb

```diff
@@ -148,44 +163,41 @@
 
     context "with lettings log" do
       context "when collection_period_open? == true" do
-        context "with 2023 deadline" do
-          let(:now) { Time.utc(2022, 6, 1) }
-          let(:lettings_log) { create(:lettings_log, :completed) }
+        let(:lettings_log) { build(:lettings_log, :completed, id: 123) }
```

Comment:
I also hadn't thought of defining id when building. Since it wasn't in the factory I thought it might not know about it which is stooopid. This is nicer than what I've done in a few cases recently